compiler/testData/codegen/box/delegatedProperty/delegateToConst/delegateToConstProperty.kt

```diff
@@ -0,0 +1,14 @@
+// WITH_STDLIB
+// WITH_REFLECT
+
+operator fun Any?.getValue(x: Any?, y: Any?): String {
+    return "OK"
+}
+const val a = 1
```

Comment:
Maybe also change the type to e.g. String in one of the tests, to check that not only integer types are supported. :)